graphql/schema/types/image.graphql

```diff
@@ -10,5 +18,16 @@ type Image {
   kernel: String!
   lastDeployed: Time!
   name: String!
+  packages(opts: PackageOpts!): [Package!]!
   versionId: String!
 }
+
+type Package {
+    name: String!
+    manager: String!
+    version: String!
+}
+
+
+
```

Comment:
nit. formatting:
- 2 spaces instead of 4 for tabs
- no extra lines at the end of the file